setup/insertObjects.js

```diff
@@ -59,8 +124,8 @@ Kompus10_2 = [{
     "Inlopp": "O: 160, typ: PVC rör, VG: 850",
     "Kabelgenomföring": "O: 50, typ: gummitätning, VG: 1800",
     "Antal pumpar": 2,
-    "Pump": "GD-20  Skärande avloppspump, 3-fas",
-    "ArtikelNr": "10-2GD20",
+    "Pump": "GD-20",
+    "Modell": "10-2GD20",
```

Comment:
This should not be named Modell.... A Modell name is already set for Kompus 10_2